### ISSUE TITLE
License fix

### ISSUE THREAD

**Issue by melg8** [nvuillam/mega-linter#376]
Hello, really like your work, i have question and sort of bug report.

# Question 
Can you, please, distribute your project under ![MIT](https://img.shields.io/badge/License-MIT-blue.svg)?
Some reasons behind that:
- First of all, that, i think would increase number of users, that can/want to use your project on github and beyond.
- Second of all, original work of [super-linter](https://github.com/github/super-linter) form which your work is derived (as stated in your readme) now is distributed with [MIT license](https://github.com/github/super-linter/blob/master/LICENSE) and for what reason some of your potential users have no other option rather than use original super-linter, even though your project is way better fit for them.
- Third - using MIT license would not impose potential risks for users which prefer to distribute theirs code under MIT license (or other permissive license), but choose to use your tools, and now have some setup code or tweaked version of your code in their codebase. Which can be interpreted as violation of terms of GPL-type license in the future. As for [example](https://github.com/mimemagicrb/mimemagic/issues/97) happened to mimemagic project, where single xml file from GPL project, even without license header - led to immense reworks in hurry just to fix and replace that code, to stay on MIT license. 

# Bug report:
Your readme file now says that project uses MIT [license](https://github.com/nvuillam/mega-linter/blob/master/README.md#license) - but link is going to AGPL-v3.0.

# Thoughts
That's funny, because, your tool is all about aggregating automatic checks, is there a tool, that can check for readme text and linked license consistency? Because if even your own code base have that issue, maybe there are other projects in the wild which have same issue.

Anyway, really appreciate your work, giving to opensource community convenient tools to develop better, more reliable, clean code.

Regards, Melg Eight.

**Comment by nvuillam** [comment]
Hi @melg8 :)

I'm far to be an expert about open-source licenses, but my goal when deciding to use AGPLV3 is to prevent the following case to happen:
- Some company likes Mega-Linter and decides to build some cloud service based on it, and **make it a paid service**
- With MIT licence they can do that
- With AGPLV3 licence, they also can do that... but they must make public the updates they would implement to build their paid service, so the open source community would be capable to use such updates without paying ^^

So basically... anyone can do what they want with MegaLinter, except making it a paid service, do you see a way to do that with MIT License ? :/

About license linter ... it seems it exists ! https://github.com/todogroup/repolinter

It could be easily integrated within MegaLinter... are you in a mood for a PR ? 😄

**Comment by ktomk** [comment]
> Bug report:
> Your readme file now says that project uses MIT license - but link is going to AGPL-v3.0.

Indeed, the link title could have been misleading, e.g. on the website. Fixed in 721eda0. Thank you for the hint.
